### DISCUSSION TITLE
My "Visitor" plugin patch is broken by v0.75

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#6535] [Self Hosting] [answered]
Asking for help !
The "visitor" plugin patch that I use since a very long time, including now with the v0.74 app seems to be broken
https://github.com/Limezy/outline_ynh/blob/v2-v0.75-anthropologiebiblique/sources/patches/main-02-visitor.patch
I can't see anywhere in the v0.74 to v0.75 commits to the code base a change that would explain this ?
There is the automatic redirect of oidc commit, but there doesn't seem to be any changes associated in the authProvider logic
If someone has an idea I would be very interested !
Thanks in advance

**Comment by tommoor** [comment]
In what sense is it broken? The OIDC redirect is the most likely corporate for sure.

**Reply by Limezy** [reply, marked as answer]
Thanks for your help ! I have solved my issue by removing the .babelrc file

**Reply by tommoor** [reply]
Great, yep - those are no longer required

**Reply by Limezy** [reply]
The Yunohost package for Outline will soon be updated to v0.75. Thanks for that great release !
